ensime_shared/launcher.py

```diff
@@ -85,9 +90,16 @@ def load_classpath(self, scala_version, java_home):
         classpath_file = os.path.join(project_dir, "classpath")
         if not os.path.exists(classpath_file):
             self.generate_classpath(scala_version, classpath_file)
-        return "{}:{}/lib/tools.jar".format(Util.read_file(classpath_file), java_home)
-
-    def start_process(self, conf_path, classpath, cache_dir, java_home, java_flags):
+        return "{}:{}/lib/tools.jar".format(
+            Util.read_file(classpath_file), java_home)
+
+    def start_process(
```

Comment:
i would change the declaration of this method, maybe two lines? line per argument is very cumbersome